packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/AuthenticationProvider.kt

```diff
@@ -27,6 +27,7 @@ import io.realm.kotlin.internal.interop.sync.AuthProvider
  */
 public enum class AuthenticationProvider(id: AuthProvider) {
     ANONYMOUS(AuthProvider.RLM_AUTH_PROVIDER_ANONYMOUS),
+    ANONYMOUS_NO_REUSE(AuthProvider.RLM_AUTH_PROVIDER_ANONYMOUS_NO_REUSE),
```

Comment:
It is a bit weird that this is its own auth provider, but I. guess we are just exposing the underlying data

Reply:
Yeah, this is Core's new enum, even though it adds no value having it 🤷 . In fact, its addition caused .NET to have a crash after they updated their Core to 12.5.0: https://github.com/realm/realm-dotnet/issues/2987 so I'd rather add it to avoid problems.

Reply:
It looks like .NET still only have one auth provider. Is there a way for us to do something similar? 🤔

Reply:
Hm, it's because they don't have an enum watchdog test but we do 😅

Reply:
I meant, is there a way for us to only have one auth provider for this in the public API?